content/bn/docs/setup/production-environment/turnkey-solutions.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Turnkey ক্লাউড সমাধান
```

Comment:
If it's not obvious how to translate “Turnkey”, try starting from “Ready made” instead?

Reply:
+1 to Ready-made

Reply:
Okay! @sftim & @shannonxtreme